<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextVerifyChange_Dialog</class>
 <widget class="QDialog" name="TextVerifyChange_Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>522</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>483</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>100000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Text verify</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>11</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Colour</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>37</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Style</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>36</y>
     <width>39</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Weight</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Layer</string>
   </property>
  </widget>
  <widget class="QLabel" name="_219">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>182</y>
     <width>57</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prompt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>214</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Text font</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>248</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>275</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>302</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Angle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>275</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vert just</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>302</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vert spacing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>248</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Horz just</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>354</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Halo Colour</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>376</y>
     <width>347</width>
     <height>102</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Style</string>
   </property>
   <widget class="QCheckBox" name="_112">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>26</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Bold</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="_114">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>49</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Italic</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="_122">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>72</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Halo text</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="_113">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>26</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Underline</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="_115">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>49</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Boxed</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="_116">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>26</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Outline</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="_120">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>49</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Scaled</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="_123">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>330</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Text height in points</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>486</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="_119">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>182</y>
     <width>289</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="_104">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>62</y>
     <width>347</width>
     <height>109</height>
    </rect>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_2">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>486</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_106">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>248</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="_109">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>246</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="_110">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>274</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_111">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>302</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <double>-100000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="_121">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>354</y>
     <width>59</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFontComboBox" name="_105">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>212</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentFont">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_107">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>274</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <double>-100000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_108">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>300</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <double>-360.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="_100">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>59</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="_102">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>36</y>
     <width>59</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="WeightSpinBox" name="_103">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>34</y>
     <width>63</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>3.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="_1000">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>9</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select Colour</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../RealCAD.qrc">
     <normaloff>:/images/PROPERTIES-colours.svg</normaloff>:/images/PROPERTIES-colours.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1002">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>35</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select Style</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../RealCAD.qrc">
     <normaloff>:/images/PROPERTIES-styles.svg</normaloff>:/images/PROPERTIES-styles.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1001">
   <property name="geometry">
    <rect>
     <x>338</x>
     <y>8</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select Layer</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../RealCAD.qrc">
     <normaloff>:/images/PROPERTIES-layers.svg</normaloff>:/images/PROPERTIES-layers.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1003">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>34</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select Weight</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../RealCAD.qrc">
     <normaloff>:/images/PROPERTIES-weights.svg</normaloff>:/images/PROPERTIES-weights.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="LayerSpinBox" name="_101">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>8</y>
     <width>149</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="maximum">
    <number>2049</number>
   </property>
  </widget>
  <widget class="QPushButton" name="_1004">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>353</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>Select Colour</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../RealCAD.qrc">
     <normaloff>:/images/PROPERTIES-colours.svg</normaloff>:/images/PROPERTIES-colours.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>_219</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_14</zorder>
  <zorder>_123</zorder>
  <zorder>_1</zorder>
  <zorder>_119</zorder>
  <zorder>_104</zorder>
  <zorder>_2</zorder>
  <zorder>_106</zorder>
  <zorder>_109</zorder>
  <zorder>_110</zorder>
  <zorder>_111</zorder>
  <zorder>_121</zorder>
  <zorder>_105</zorder>
  <zorder>_107</zorder>
  <zorder>_108</zorder>
  <zorder>_100</zorder>
  <zorder>_102</zorder>
  <zorder>_103</zorder>
  <zorder>_1000</zorder>
  <zorder>_1002</zorder>
  <zorder>_1001</zorder>
  <zorder>_1003</zorder>
  <zorder>_101</zorder>
  <zorder>_1004</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RCDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../RealCADQt/source/rcdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>WeightSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../RealCADQt/source/weightspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>LayerSpinBox</class>
   <extends>QSpinBox</extends>
   <header>../RealCADQt/source/layerspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../RealCAD.qrc"/>
 </resources>
 <connections/>
</ui>
